### ISSUE TITLE
How to use in a table?

### ISSUE THREAD

**Issue by james-ingold** [tangbc/vue-virtual-scroll-list#299]
Trying to figure out how to get this to work in a table: 

The html is like this: 

```
<table>
<thead></thead>
<tbody>
<!-- scrollable virtual list -->
<VirtualList
:data-component="aTRcomponent">
</VirtualList>
</tbody>
</table>
```

aTRComponent pseudo code:
```
<template>
<tr></tr>
</template>
```

**Comment by Flavsditz** [comment]
I am using the same way.

I had to do a lot of changes to my components such as adding the "source" prop and so on. 
I think this could be described better in the documentation, i.e. How should the data-component look like.

I made it work however there are some things I don't like or understand:

- I've added the `page-mode` since I wanted the scrollbar of my existing table to be used
- The object gets a couple of wrapper DIV that breaks up my existing design:
-- Each item gets a div (`role="listitem"`)
-- The whole thing gets a div of a group 
-- Finally the group also gets wrapped in a div
- I only get 30 items that get rendered. 
-- This I can increase using the `keeps` prop
-- I still can't see the others when I scroll
